doc/material-colors.md

```diff
@@ -50,7 +50,7 @@ uid: Uno.Themes.Material.Colors
 | FocusedOpacity  | 0.12  |
 | PressedOpacity  | 0.12  |
 | DraggedOpacity  | 0.16  |
-| SelectedOpacity | 0.16  |
+| SelectedOpacity | 0.08  |
```

Comment:
@kazo0, @Xiaoy312 this change is not a breaking change but should we mention something in the migration guide documentation?
We added some details for v5 for the previous changes: https://github.com/unoplatform/Uno.Themes/blob/85732f91e2d214c177533cc54b975e687622b084/doc/material-migration.md?plain=1#L90-L98